app/containers/App/Sidebar/NetworkSwitch/index.js

```diff
@@ -15,11 +15,12 @@ const mapActionsToProps = (actions: Actions): Object => ({
 })
 
 const mapAccountActionsToProps = (actions, props) => ({
-  loadWalletData: () =>
+  loadWalletData: (net: string) =>
```

Comment:
Possibly a bug in spunky but even though the state tree was being updated succesfully the actions below were being called with the previous state 😅 this change maps the change event from the select input to the invocation of these actions